src/janetsh

```diff
@@ -60,9 +60,21 @@
 (array/remove process/args 0 (length process/args))
 (array/concat process/args new-args)
 
+(defn cwd-tilded
+  "Returns current directory with ~ in place of $HOME."
+  []
+  (let [cwd (os/cwd)
+        home (os/getenv "HOME")
+        dir (cond
+              (string/has-prefix? (string home "/") cwd)
+              (string/replace home "~" cwd)
```

Comment:
String replace is a bug - you need to only replace the first tild.

 I think this code can be shortened a tad using an or and string/slice, you only need two conditions.